src/Valit/Rules/Extensions/ValitRuleByteExtensions.cs

```diff
@@ -38,7 +38,49 @@ public static class ValitRuleByteExtensions
         public static IValitRule<TObject, byte> IsNonZero<TObject>(this IValitRule<TObject, byte> rule) where TObject : class
         {
             rule.ThrowIfNull(ValitExceptionMessages.NullRule);    
-            return rule.Satisfies(p => p != 0);
+            return rule.Satisfies(p =>  p != 0);
+        }
+
+        public static IValitRule<TObject, byte?> IsGreaterThan<TObject>(this IValitRule<TObject, byte?> rule, byte value) where TObject : class
```

Comment:
Imho we should put nullable implementations right after non- nullable so it's easier to find something.

So something like:
IsGreaterThan(this IValitRule<TObject, int>) rule....)
IsGreaterThan(this IValitRule<TObject, int?>) rule....)